server/interceptor/access/access.go

```diff
@@ -37,11 +37,11 @@ func Intercept(w http.ResponseWriter, r *http.Request) error {
 
 	r.Body = http.MaxBytesReader(w, r.Body, core.ServerInfo.Config.MaxBodyBytes)
 
-	if !validate.IsRequestURI(r.RequestURI) {
-		err := fmt.Errorf("Invalid Request URI %s", r.RequestURI)
+	/*if !validate.IsRequestURI(r.RequestURI) {
```

Comment:
Why disenable this check? Is there some conflict?

Reply:
When retrieving pacts (at the provider side) using pact jvm library, it fires a base url "/". I have provided GetHome API for serving this request, however currently service-center blocks this request. Any suggestions?